index.css

```diff
@@ -0,0 +1,73 @@
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-family: 'Lato', sans-serif;
+}
+
+.contact-form {
+  max-width: 800px;
+}
+
+.contact-form .cf-item {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  font-size: 1.6rem;
+}
+
+.contact-form .cf-input {
+  padding: 12px;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+  margin-top: 6px;
+  margin-bottom: 16px;
+  resize: vertical;
+}
+
+.cf-functionality-wrap {
+  display: flex;
+  flex-wrap: wrap;
+  border-radius: 4px;
+  box-sizing: border-box;
+  border: 1px solid #ccc;
+  padding: 40px;
+  margin-bottom: 10px;
+}
+
+.cf-functionality-wrap .cf-item {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  flex-grow: 1;
+  flex-shrink: 1;
+  margin: 10px;
+}
+
+.cf-functionality-wrap .cf-item .cf-input {
+  margin: 0 auto;
+}
+
+.contact-form .cf-error {
```

Comment:
I'm finding it difficult to see which error is associated with which input. Consider spacing things out differently and putting error messages closer to their respective input.